app/controllers/api/v1/account_requests_controller.rb

```diff
@@ -0,0 +1,84 @@
+class Api::V1::AccountRequestsController < ApplicationController
+
+  # GET /pending_request
+  def pending_request
+    @account_requests = AccountRequest.where(status: 'Under Review').order('created_at DESC')
+    render json: @account_requests, status: :ok
+  end
+
+  # GET /processed_request
+  def processed_request
+    @account_requests = AccountRequest.where.not(status: 'Under Review').order('updated_at DESC')
+    render json: @account_requests, status: :ok
+  end
+
+  # POST /account_requests
+  def create
+    @account_request = AccountRequest.new(account_request_params)
+    if @account_request.save
+      render json: @account_request, status: :created
+    else
+      render json: @account_request.errors, status: :unprocessable_entity
+    end
+  rescue StandardError => e
+    render json: { error: e.message }, status: :unprocessable_entity
+  end
+
+  # GET /account_requests/:id
+  def show
+    @account_request = AccountRequest.find(params[:id])
+    render json: @account_request, status: :ok
+  rescue ActiveRecord::RecordNotFound => e
+    render json: { error: e.message }, status: :not_found
+  end
+
+  # PATCH/PUT /account_requests/:id
+  def update
+    @account_request = AccountRequest.find(params[:id])
+    @account_request.update(account_request_params)
+    if @account_request.status == 'Approved'
+      create_approved_user
+    else
+      render json: @account_request, status: :ok
+    end
+  rescue StandardError => e
```

Comment:
It will always give`unprocessable_entity` but what if the record is not found at line 37
You can even add `rescue_from` at the top to handle this common `RecordNotFound`